erpnext/stock/doctype/batch/batch_dashboard.py

```diff
@@ -13,7 +13,7 @@ def get_data():
 			},
 			{
 				'label': _('Sell'),
-				'items': ['Sales Invoice', 'Delivery Note']
+				'items': ['Sales Invoice', 'Delivery Note', 'Sales Order']
```

Comment:
reset this to ["Sales Order", "Sales Invoice", "Delivery Note"] - in that sequence.